src/borg/archiver.py

```diff
@@ -1171,6 +1172,7 @@ def _delete_archives(self, args, repository):
 
         if args.forced == 2:
             deleted = False
+            _logger = logging.getLogger('borg.output.list') if self.output_list else logger
```

Comment:
well, i recommended this if/else some reviews ago, but meanwhile i noticed that the --list output is just *additional* output going to that logger *only* (and no output in the other case).

see other places where it uses borg.output.list.